configure.py

```diff
@@ -1198,6 +1228,8 @@ def configure_node(o):
     configure_arm(o)
   elif target_arch in ('mips', 'mipsel', 'mips64el'):
     configure_mips(o, target_arch)
+  elif sys.platform == 'zos':
+    configure_static_zoslib(o)
```

Comment:
Should this be `configure_zos` to be consistent with the other platforms. It also would make sense if we ever need more configuration for zos and add it to the function.